src/core/Akka.Remote.Tests/Transport/AkkaProtocolStressTest.cs

```diff
@@ -178,7 +178,7 @@ public AkkaProtocolStressTest() : base(AkkaProtocolStressTestConfig)
 
         #region Tests
 
-        [Fact(Skip = "fails due to out-of-order processing as a result of Helios eventing")]
+        [Fact]
         public void AkkaProtocolTransport_must_guarantee_at_most_once_delivery_and_message_ordering_despite_packet_loss()
         {
```

Comment:
This still fails, wasn't message order for Helios solved some time ago? (fixed by limiting worker threads IIRC?)
Will revert this change if this is still an issue

cc @Aaronontheweb